Formula/qgis3-dev.rb

```diff
@@ -216,6 +216,7 @@ def install
       flex
       bison
       fcgi
+      libzip
```

Comment:
`libzip` is already listed 4 lines above.

Reply:
oops, removing.